pkgs/applications/kde/dragon.nix

```diff
@@ -0,0 +1,25 @@
+{
+  mkDerivation, lib,
+  extra-cmake-modules, kdoctools,
+  baloo, baloo-widgets, kactivities, kbookmarks, kcmutils,
+  kcompletion, kconfig, kcoreaddons, kdelibs4support, kdbusaddons,
+  kfilemetadata, ki18n, kiconthemes, kinit, kio, knewstuff, knotifications,
+  kparts, ktexteditor, kwindowsystem, phonon, solid, phonon-backend-gstreamer
+}:
+
+mkDerivation {
+  name = "dragon";
+  meta = {
+    license = with lib.licenses; [ gpl2 fdl12 ];
+    description = "A simple media player for KDE";
+    maintainers = [ lib.maintainers.ttuegel ];
```

Comment:
You probably don't want to add @ttuegel as a maintainer :)

Reply:
You're probably right. That was just copypasta. Do I add myself as a maintainer, and add that to this PR?

Reply:
Just added myself as maintainer, and changes this line.